packages/odyssey-react/src/components/Infobox/index.tsx

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+import type { FunctionComponent, ReactNode } from 'react';
+import { useCx, useOmit } from '../../utils';
+
+export type InfoboxVariants = 'info' | 'danger' | 'caution' | 'success';
+
+export type Props = {
+  /**
+   * Content to be rendered within the infobox. Avoid using direct children, put child content
+   * within the provided Infobox static components (Infobox.Content and Infobox.Actions)
+   */
+  children: ReactNode,
+  
+  /**
+   * The visual variant to be displayed to the user.
+   * @default info
+   */
+  variant?: InfoboxVariants,
+
+  /**
+   * The title or headline of the Infobox. If Infobox.Content is not present it is required.
+   */
+  title?: string;
+};
+
+type PropsInfoboxContent = {  
+  children: ReactNode
+}
+
+type PropsInfoboxActions = {  
+  children: ReactNode
+}
+
+export type StaticComponents = {
+  Content: FunctionComponent<PropsInfoboxContent>,
+  Actions: FunctionComponent<PropsInfoboxActions>,
+}
+
+/**
+ * An infobox is a type of alert that provides feedback in response to a 
+ * user action or system activity.
+ *
+ * @component
+ * @example
+ * <Infobox variant="primary" title="Moonbase Alpha-6">
+ *  <Infobox.Content>
+ *    You are currently logged in from Moonbase Alpha-6, located on Luna.
+ *  </Infobox.Content>
+ *  <Infobox.Actions>
+ *    <Link href="/fuel">Visit fueling console</Link>
+ *  </Infobox.Actions>
+ * </Infobox>
+ */
+
+ const Infobox: FunctionComponent<Props> & StaticComponents = (props) => {
+  const {
+    children,
+    title,
+    variant = "info",
+    ...rest
+  } = props;
+
+  const componentClass = useCx(
+    "ods-infobox",
+    variant && `is-ods-infobox-${variant}`
+  );
+
+  const omitProps = useOmit(rest);
+
+  return (
+    <aside
+      className={componentClass}
+      role="status"
+      {...omitProps}
+    >
+      <span className="ods-infobox--icon">
+        {/* @todo Insert <Icon> component */}
+        ‽
```

Comment:
what is this character?

Reply:
It's an [Interrobang](https://en.wikipedia.org/wiki/Interrobang), meant to be a visual placeholder for an Icon. If it bugs, I'm happy to remove it -- I don't feel strongly about it.